gateway/host_checker_manager_test.go

```diff
@@ -0,0 +1,193 @@
+package gateway
+
+import (
+	"bytes"
+	"context"
+	"net/http"
+	"testing"
+	"text/template"
+
+	"github.com/TykTechnologies/tyk/config"
+	"github.com/TykTechnologies/tyk/storage"
+	uuid "github.com/satori/go.uuid"
+)
+
+func TestHostCheckerManagerInit(t *testing.T) {
+	ts := StartTest()
+	defer ts.Close()
```

Comment:
```suggestion
```

Why start this and not use it at all. Redis will be up and running already before `Main` function. You can do redis stuff fine.